<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 63 9pt &quot;Inter 18pt SemiBold&quot;;
    background-color: #100917;
color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignTop">
    <widget class="QWidget" name="local_cadastro" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QLabel{
	padding: 0px 2px;
	max-height:20px;
 	margin-top: 2px;
}

QLineEdit{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	max-height: 40px;   /* largura mínima */
	max-width: 500px;   /* largura mínima */
	min-height:25px;
    border-radius: 6px;
	border: none;
}
Line{
	background-color: rgb(255, 255, 255);
	max-height:2px;
}
#local_cadastro{
	
	
	
    border-radius: 6px;
    border: 3px solid #5a5a5a;

}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lbl_nome">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Nome do Cargo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_funcao">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Função exercida...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="separacao_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="lbl_email">
           <property name="text">
            <string>Permissão do Cargo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_permissoes">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>506</width>
             <height>41</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="focusPolicy">
            <enum>Qt::WheelFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">	color: #000000;
	background-color: rgb(255, 255, 255);
	max-height: 35px;   /* largura mínima */
	max-width: 500px;   /* largura mínima */
    border-radius: 6px;
	border: none;
    border-radius: 6px;
    border: 3px solid #ffffff;
  </string>
           </property>
           <item>
            <property name="text">
             <string>Usuário</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Administrador</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="btn_adicionar_cargo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>114</width>
          <height>36</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 63 12pt &quot;Inter 18pt SemiBold&quot;;
background-color: rgb(83, 131, 195);
border: 3px solid #2196F3;
color: white;
    border-radius: 6px;</string>
        </property>
        <property name="text">
         <string>Adicionar</string>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
